### PR TITLE
Add component by using prop

### PR DESCRIPTION
**Pull request by francksefu**
Create a new directory called components. -> done
Inside components, create a new Calculator.js file. -> done
In Calculator.js, create a React component  -> done

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  <button type="button" className={classons}>
+    {valeur}
+  </button>
+);
+
+Span.propTypes = {
+  classons: PropTypes.string.isRequired,
+  valeur: PropTypes.string.isRequired,
+};
+
```
Comment (DammyShittu) [inline src/components/Calculator.js:53]: - **[OPTIONAL]** It would be better to create a new file for this component for a more organized codebase

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @francksefu,

## Highlights
✔️ Functional component

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Kindly remove the following files.

  fkori
  mdkdie

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (francksefu) [reply src/components/Calculator.js:53]: Okay Thanks, I will do that in the next challenge

**Review by DammyShittu — APPROVED** [review]
Hi @francksefu,

## APPROVED 🟢

## Highlights
✔️ Functional components

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**